src/sources/socket/mod.rs

```diff
@@ -383,6 +381,53 @@ mod test {
         SourceSender,
     };
 
+    fn get_gelf_payload(message: &str) -> String {
```

Comment:
I don't know if those test utility methods should be defined here. Didn't find anywhere else to put them.

Reply:
👍 this is inline with existing patterns in the codebase.